TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -55,6 +55,12 @@ public class Workspace
         public EventHandler<string> WarningTrigger { get; set; }
         public Queue<MessageActionWrapper> MessagesActionsQueue { get; private set; }
         private Func<string, Uri, bool> _Logger;
+        public List<string> CustomAnalyzerFiles { get; set; }
+        /// <summary>
+        /// Custom Analyzer Providers Loaded
+        /// </summary>
+        private IEnumerable<IAnalyzerProvider> _customAnalyzerProviders;
```

Comment:
We should store an array of `IAnalyzerProvider`, meaning an already enumerated collection, see my remark below.

Reply:
done in 78ea44d